instrumentation/manticore/lib/opentelemetry/instrumentation/manticore/patches/response.rb

```diff
@@ -0,0 +1,104 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+module OpenTelemetry
+  module Instrumentation
+    module Manticore
+      module Patches
+        # Module to prepend to RestClient::Request for instrumentation
+        module Response
+          # Hooks into the #.call function which is executed for any verbs like:
+          #   [:get, :post, :put, :options, :header]
+          # Executes for async and sync requests.
```

Comment:
small request, I'd like to see a test added for an async request as well, both to make sure we are successfully tracing an async request and also to make sure that multiple async requests don't mix context when executing at the same time. I'm not familiar with this library or how it handles async requests so i'd just like to make sure we have some tests to ensure we dont have any regressions here, if that's possible.